data/presets/power/switchgear.json

```diff
@@ -0,0 +1,14 @@
+{
+    "icon": "temaki-power",
```

Comment:
what about an icon like `temaki-power_switch` which would represent the _switching_ aspect of these facilities?